devicon.json

```diff
@@ -3752,6 +3752,22 @@
         },
         "color": "#D1A41A",
         "aliases": []
+    },
+	{
+        "name": "tortoisegit",
+        "tags": [
+            "git"
+        ],
+        "versions": {
+            "svg": [
+                "original",
```

Comment:
Remove the `,` here please.

PS: this is valid syntax for JS for JSON is unnecessary strict here. Took me a while to find this.